controllers/articles.js

```diff
@@ -4,7 +4,6 @@ import models from '../models/index';
 const { article, User } = models;
 
 const articles = {
-
   /*
    * Deleting a post based to its slug.
```

Comment:
The correct preposition to use here is `on`. 
Edit the sentence to say, "Deleting a post based on its slug".